src/drivers/ulanding/ulanding.cpp

```diff
@@ -92,6 +149,16 @@ Radar	*g_dev;
 }
 
 Radar::Radar(const char *port) :
+#if defined(__PX4_POSIX)
+	VDev("Radar", RANGE_FINDER0_DEVICE_PATH),
```

Comment:
Isn't this identical either than VDev/CDev?

Reply:
@jgs2185 should this different be kept also?

Reply:
If you're asking about the VDev/CDev difference, yes, it needs to be there.